app/models/short_submission.rb

```diff
@@ -30,8 +30,8 @@ class ShortSubmission < ActiveRecord::Base
   before_validation :remove_leading_zeroes
 
   private def remove_leading_zeroes
-    return if answer.nil?
-
+    return if answer.blank?
+    
```

Comment:
This is the line that caused the trailing whitespace cop problem. Can you remove the spaces in this line?

Reply:
sorry for my reckless